### PR TITLE
Review changes

### PR DESCRIPTION
**Pull request by eesmerdag**

### CODE REVIEW

**Comment by eesmerdag** [comment]
1. Correct data fetching from smart contract for getNode method; The order is not correct for fetching node and should be corrected.
2. ValidatorStatisticsTypeFee is not updatable, so SysEvtTypeFeeChanged is not needed.
3. Delegations table created_at index

File: scraper/structs/validator_statistics.go [hunk 1]
```diff
@@ -32,7 +31,6 @@ var (
 		"ACTIVE_NODES":      ValidatorStatisticsTypeActiveNodes,
 		"LINKED_NODES":      ValidatorStatisticsTypeLinkedNodes,
 		"MDR":               ValidatorStatisticsTypeMDR,
-		"FEE":               ValidatorStatisticsTypeFee,
```
Comment (lukanus) [inline scraper/structs/validator_statistics.go:35]: Keep it.

File: api/skale/node.go [hunk 2]
```diff
@@ -157,7 +157,7 @@ func (c *Caller) GetNode(ctx context.Context, bc *bind.BoundContract, blockNumbe
 
 	lrDate := results[5].(*big.Int)
 	IP := results[1].([4]byte)
-	publicIP := results[1].([4]byte)
+	publicIP := results[2].([4]byte)
```
Comment (lukanus) [inline api/skale/node.go:160]: 👍

File: client/actions/actions.go [hunk 3]
```diff
@@ -590,12 +590,7 @@ func (m *Manager) AfterEventLog(ctx context.Context, c contract.ContractsContent
 
 func (m *Manager) saveValidatorStatChanges(ctx context.Context, validator structs.Validator, blockNumber uint64, time time.Time) error {
 
-	err := m.dataStore.SaveValidatorStatistic(ctx, validator.ValidatorID, blockNumber, time, structs.ValidatorStatisticsTypeFee, validator.FeeRate)
```
Comment (lukanus) [inline client/actions/actions.go:593]: Keep it #2

File: store/postgresql/validator.go [hunk 4]
```diff
@@ -110,11 +110,6 @@ func (d *Driver) GetValidators(ctx context.Context, params structs.ValidatorPara
 		args = append(args, params.ValidatorID)
 		i++
 	}
-	if params.Active != "" {
```
Comment (lukanus) [inline store/postgresql/validator.go:113]: Is there a reason why we don't need this? (just asking)

Comment (eesmerdag) [reply store/postgresql/validator.go:113]: i remember i added that parameter but it is not used at all. it is just for getting authorized validators but they should be filtered on front end

Comment (lukanus) [reply store/postgresql/validator.go:113]: 👍

**Review by lukanus — APPROVED** [review]

**Pull request merged**